source/renderer/app/components/voting/voting-registration-wizard-steps/VotingRegistrationStepsRegister.tsx

```diff
@@ -186,15 +184,9 @@ class VotingRegistrationStepsRegister extends Component<Props> {
         actions={actions}
         onBack={onBack}
         containerClassName={styles.component}
-        nextFundNumber={nextFundNumber}
       >
         <p className={styles.description}>
-          <FormattedHTMLMessage
-            {...messages.description}
-            values={{
-              nextVotingFundNumber: nextFundNumber,
-            }}
-          />
+          <FormattedHTMLMessage {...messages.description} />
```

Comment:
Same as previous comment, `intl.formatMessage(messages.description);` ?

Reply:
Resolved in c076a3917e896a89cfb497d66c21828eacb5ecd3, good catch!